fhevm/tfhe.go

```diff
@@ -18,9 +18,12 @@ package fhevm
 
 /*
 #cgo linux CFLAGS: -O3 -I../tfhe-rs/target/release
-#cgo linux LDFLAGS: -L../tfhe-rs/target/release -l:libtfhe.a -lm
+#cgo linux CFLAGS: -O3 -I../tfhe-rs/target/release/deps
+#cgo linux LDFLAGS: -L../tfhe-rs/target/release -l:libtfhe.a -L../tfhe-rs/target/release/deps -l:libtfhe_c_api_dynamic_buffer.a  -lm
 #cgo darwin CFLAGS: -O3 -I../tfhe-rs/target/release
+#cgo darwin CFLAGS: -O3 -I../tfhe-rs/target/release/deps
 #cgo darwin LDFLAGS: -framework Security -L../tfhe-rs/target/release -ltfhe -lm
+#cgo darwin LDFLAGS: -framework Security -L../tfhe-rs/target/release/deps -ltfhe_c_api_dynamic_buffer
```

Comment:
here the you can copy the end of the line after -framework Security and put it in the line above I think